cg/store/models.py

```diff
@@ -999,3 +999,17 @@ class SampleRunMetrics(Base):
     __mapper_args__ = {
         "polymorphic_on": "type",
     }
+
+
+class IlluminaSampleRunMetrics(SampleRunMetrics):
+    __tablename__ = "illumina_sample_run_metrics"
+
+    id: Mapped[int] = mapped_column(ForeignKey(SampleRunMetrics.id), primary_key=True)
+    flow_cell_lane: Mapped[int | None]
+    total_reads_in_lane: Mapped[BigInt | None]
+    base_passing_q30_percent: Mapped[Num_6_2 | None]
+    base_mean_quality_score: Mapped[Num_6_2 | None]
+    created_at: Mapped[datetime | None]
+    __mapper_args__ = {
+        "polymorphic_identity": "illumina_sample_run_metrics",
```

Comment:
I think this should be
```suggestion
        "polymorphic_identity": DeviceType.ILLUMINA
```
From the docs:

> In the above example, each mapping specifies the [Mapper.polymorphic_identity](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.Mapper.params.polymorphic_identity) parameter within its mapper arguments. This value populates the column designated by the [Mapper.polymorphic_on](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.Mapper.params.polymorphic_on) parameter established on the base mapper.In the above example, each mapping specifies the [Mapper.polymorphic_identity](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.Mapper.params.polymorphic_identity) parameter within its mapper arguments. This value populates the column designated by the [Mapper.polymorphic_on](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.Mapper.params.polymorphic_on) parameter established on the base mapper.